soccer/ui/MainWindow.ui

```diff
@@ -1559,6 +1559,16 @@ p, li { white-space: pre-wrap; }
                   </widget>
                  </item>
                  <item>
+                  <widget class="QCheckBox" name="useDefenseCheckBox">
```

Comment:
Yeah that is a good idea. I'll look into it, I think it would require messing around in rootplay but it shouldn't be too hard. Also it does stop sometimes go on/off automatically when you check/uncheck the box, but that is sporadic and I don't know why it happens.